plugin/src/test/resources/ide/language/xml/Example.html

```diff
@@ -5,8 +5,8 @@
     <title><warning descr="MORFOLOGIK_RULE_EN_US">Ttle</warning></title>
 </head>
 <body>
-<H1>Example <warning descr="MORFOLOGIK_RULE_EN_US">heder</warning> with <warning descr="EN_A_VS_AN">a</warning> error</H1>\
-<tag descr="it <warning descr="IT_VBZ">are</warning> error in tag <warning descr="MORFOLOGIK_RULE_EN_US">atribute</warning>"></tag>
+<H1>Example <warning descr="MORFOLOGIK_RULE_EN_US">hedder</warning> with <warning descr="EN_A_VS_AN">a</warning> error</H1>\
```

Comment:
Hm?

Reply:
We enabling 16 languages now, so some words with typos in tests exist in this languages :)